src/entities/BlockNode/index.ts

```diff
@@ -0,0 +1,65 @@
+import { EditorDocument } from '../EditorDocument';
+import { BlockTune, BlockTuneName } from '../BlockTune';
+import {
+  BlockNodeConstructorParameters,
+  BlockNodeName,
+  createBlockNodeName,
+  DataKey,
+  createDataKey, BlockNodeChildren
```

Comment:
look like the `types` folder is not the correct naming, since `create...` are not types